6.x/custom-validation-rules.md

```diff
@@ -10,61 +10,37 @@ Some Backpack fields are more difficult to validate using standard Laravel valid
 <a name="valid-upload-validation-rule"></a>
 ## `ValidUpload` for `upload` field type
 
-The `ValidUpload` rule is used to validate the `upload` field type. Using the custom rule helps developer avoid to setting different validation rules for different operations.(create/update).
+The `ValidUpload` rule is used to validate the `upload` field type. Using the custom rule helps developer avoid to setting different validation rules for different operations, (create/update).
 
 ```php
 // for the field
 CRUD::field('avatar')->type('upload');
 
 // you can write the validation rule as
-'avatar' => ValidUpload::fileRules(File::types(['jpg','png'])->max(2048))->attributeRules('required'),
+
+use Backpack\CRUD\app\Library\Validation\Rules\ValidUpload;
+
+'avatar' => ValidUpload::field('required')->file(File::types(['jpg','png'])->max(2048)),
```

Comment:
Doesn't this do the same, but look cleaner? Otherwise at first I thought you can do `max()` on the ValidUpload itself 🤦‍♂️ I think people are going to just _assume_ they can use class syntax for validation too, since it's a Laravel thing, not a Backpack thing: 

```suggestion
'avatar' => ValidUpload::field('required')->file('mimes:jpg,png|max:2048'),
```